core/src/com/unciv/models/ruleset/Building.kt

```diff
@@ -48,6 +48,7 @@ class Building : RulesetStatsObject(), INonPerpetualConstruction {
     var isWonder = false
     var isNationalWonder = false
     fun isAnyWonder() = isWonder || isNationalWonder
+    @Deprecated("Since 3.18.15 - replaced with RequiresAnotherBuilding unique")
     var requiredBuilding: String? = null
```

Comment:
Why not remove this outright, if its uses have been removed?

Reply:
Tru dat